docs/releases/1.10.txt

```diff
@@ -214,6 +214,13 @@ Minor features
   ``{% load staticfiles %}`` or ``{% load static from staticfiles %}``) and
   not worry about whether or not the ``staticfiles`` app is installed.
 
+:mod:`django.utils.feedgenerator`
```

Comment:
Since 1.10a1 has already been released and this patch doesn't fix a bug (it's rather a cleanup or a minor feature), I think it needs to go into 1.11. See https://docs.djangoproject.com/en/dev/internals/release-process/#release-cycle for details.

> Should I put it in 1.11.txt and remove from 1.10.txt?

Yes.